thirdeye-ui/src/app/components/app-bar/app-bar.component.tsx

```diff
@@ -92,6 +93,18 @@ export const AppBar: FunctionComponent = () => {
                     </NavBarLinkIconV1>
                     <NavBarLinkTextV1>{t("label.anomalies")}</NavBarLinkTextV1>
                 </NavBarLinkV1>
+                {/* Impact Dashboard */}
+                {/* <NavBarLinkV1
+                    href={AppRoute.IMPACT_DASHBOARD}
+                    selected={isRouteCurrent(AppRoute.IMPACT_DASHBOARD)}
+                >
+                    <NavBarLinkIconV1>
+                        <MultilineChartIcon />
+                    </NavBarLinkIconV1>
+                    <NavBarLinkTextV1>
+                        {t("pages.impact-dashboard.title")}
+                    </NavBarLinkTextV1>
+                </NavBarLinkV1> */}
```

Comment:
Let's remove these?

Reply:
I will be uncommenting these after the design review. Till then we don't want it present in left-nav. hence commented it out